test/autest/gold_tests/autest-site/txn_box.test.ext

```diff
@@ -54,60 +54,57 @@ ExtendTestRun(ConfigureTsForTxnBox, name="ConfigureTsForTxnBox")
 
 
 def TxnBoxTestRun(self, text, replay_path, **kw):
-  """
-  Set up a standard test run for TxnBox
+    """
+    Set up a standard test run for TxnBox
 
-  Args:
-    text: Description for test run.
-    replay_path: Path to the replay file.
-  Keywords
-    replay: The replay file.
-  """
+    Args:
+        text: (str) Description for test run.
+        replay_path: (str) Path to the replay file.
+    Keywords
+        replay_path: (str) The replay file.
+    """
 
-  config_key = kw.get("config_key")
-  config_path = kw.get("config_path", replay_path)
-  run = self.AddTestRun(text)
+    config_key = kw.get("config_key")
+    config_path = kw.get("config_path", replay_path)
+    run = self.AddTestRun(text)
 
-  ts = self.MakeATSProcess("ts")
-  run.Variables.TS = ts;
-  pv_client = run.AddVerifierClientProcess(
```

Comment:
Is the client process no longer needed, or is it added elsewhere?

Reply:
I put it back. It was never "used" so Python complained about it.

I've added a `StartBefore` for it so that the server is explicitly started before the client.